### PR TITLE
Updated EG test list

### PR DESCRIPTION
**Pull request by agrabeli**
I carried out some research to identify further URLs to add to the Egyptian test list. This pull request includes URLs that may be interesting to test, as well as URLs that have previously been reported to be blocked.

### CODE REVIEW

File: lists/eg.csv [hunk 1]
```diff
@@ -674,3 +674,148 @@ http://www.gardennetworks.com,ANON,Anonymization and circumvention tools,2017-02
...
+http://greenarm.org/,ENV,Environment,2018-02-09,OONI,
+http://www.ape.org.eg/,ENV,Environment,2018-02-09,OONI,
+http://web.cedare.org/,ENV,Environment,2018-02-09,OONI,
+http://www.eaee-eg.com/,ENV,Environment,2018-02-09,OONI,
+http://www.ape-egypt.com/,ENV,Environment,2018-02-09,OONI,
+https://bedayaalgbt.wordpress.com/,LGBT,LGBT,2018-02-10,OONI,
+https://solidaritywithegyptlgbt.wordpress.com/,LGBT,LGBT,2018-02-10,OONI,
+https://alborsanews.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://elbadil.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://www.aljazeera.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
```
Comment (hellais) [inline lists/eg.csv]: It seems like we have aljazeera.net in the global list. We should consider updating the global list to have the .com as well.

File: lists/eg.csv [hunk 2]
```diff
@@ -674,3 +674,148 @@ http://www.gardennetworks.com,ANON,Anonymization and circumvention tools,2017-02
...
+http://www.watantvlive.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
+https://www.al-sharq.com/,NEWS,News Media,2018-02-10,OONI,
+https://www.madamasr.com/,NEWS,News Media,2018-02-10,OONI,Reportedly blocked
```
Comment (hellais) [inline lists/eg.csv]: This is a duplicate (it's present already without the `www.`)

File: lists/eg.csv [hunk 3]
```diff
@@ -674,3 +674,148 @@ http://www.gardennetworks.com,ANON,Anonymization and circumvention tools,2017-02
...
+http://www.dakahliaikhwan.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://egyptdailynews.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+https://egyptwindow.net/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://elbehira.net/elbehira/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://elwehda.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://fj-p.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://gharbiaonline.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://cluw.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+http://anhri.net/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+https://www.hrw.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
```
Comment (hellais) [inline lists/eg.csv]: This is already present in the global test list and some paths of the https site are already present in the EG list. It should be dropped.

File: lists/eg.csv [hunk 4]
```diff
@@ -674,3 +674,148 @@ http://www.gardennetworks.com,ANON,Anonymization and circumvention tools,2017-02
...
+http://jatoeg.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+https://rsf.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+https://timep.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+http://tiranwsanafir.com/,POLR,Political Criticism,2018-02-12,OONI,
+https://avaaz.org/,POLR,Political Criticism,2018-02-12,OONI,
```
Comment (hellais) [inline lists/eg.csv]: Present already in the global list.

File: lists/eg.csv [hunk 5]
```diff
@@ -674,3 +674,148 @@ http://www.gardennetworks.com,ANON,Anonymization and circumvention tools,2017-02
...
+https://egyptwindow.net/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://elbehira.net/elbehira/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://elwehda.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://fj-p.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://gharbiaonline.com/,NEWS,News Media,2018-02-12,OONI,Reportedly blocked
+http://cluw.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+http://anhri.net/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+https://www.hrw.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+http://jatoeg.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
+https://rsf.org/,HUMR,Human Rights Issues,2018-02-12,OONI,Reportedly blocked
```
Comment (hellais) [inline lists/eg.csv]: Present already in the global list.

**Comment by agrabeli** [comment]
@hellais thanks for the review, I have removed URLs that are already included in the global list.

**Comment by hellais** [comment]
@agrabeli this PR is now out of sync with master. Can you look into resolving the conflicts and ensuring there are no dupes anymore?

**Comment by agrabeli** [comment]
I'm closing this PR as I have moved all the URLs from this PR to #336.

**Pull request closed without merging**